library/kani_macros/src/lib.rs

```diff
@@ -126,7 +127,29 @@ mod sysroot {
         };
     }
 
+    struct ProofOptions {
+        schedule: Option<syn::Expr>,
+    }
+
+    impl Parse for ProofOptions {
+        fn parse(input: ParseStream) -> syn::Result<Self> {
+            if input.is_empty() {
+                Ok(ProofOptions { schedule: None })
+            } else {
+                let ident = input.parse::<syn::Ident>()?;
+                assert_eq!(
```

Comment:
Can you please turn this into a user friendly error?

You can try using `proc_macro_error`. We use it here:
https://github.com/model-checking/kani/blob/bb53ec2e4eb6937a5d45774d73307d74c74aa82d/library/kani_macros/src/derive.rs#L70